components/fieldgroups/campaign/campaign-components.schema.json

```diff
@@ -12,7 +12,7 @@
   "meta:extensible": true,
   "meta:abstract": true,
   "meta:intendedToExtend": ["https://ns.adobe.com/xdm/classes/campaign"],
-  "description": "This mixin is used to capture b2b campaign specific relationship attributes that are represented as it is in source and also used for merge/unmerge rules analysis if and when needed.",
+  "description": "This field group is used to capture b2b campaign specific relationship attributes that are represented as it is in source and also used for merge/unmerge rules analysis if and when needed.",
```

Comment:
"Captures B2B campaign..."

Let's capitalize B2B wherever used to stay consistent.